api/Sources/Api/Services/Websocket/AppConnection.swift

```diff
@@ -12,10 +12,15 @@ actor AppConnection {
     let keychains: [Keychain.Id]
   }
 
+  enum FilterStateData {
```

Comment:
this enum is how we are bridging the two types of data being reported by pre 2.7 and post. it will eventually go away.